jobs/payment-jobs/tasks/eft_task.py

```diff
@@ -90,6 +92,9 @@ class EFTCILRollup:
             .filter(InvoiceModel.total == cil_rollup.c.rollup_amount)
 
         match status:
+            case EFTCreditInvoiceStatus.CANCELLED.value:
```

Comment:
May need to think about this section bit, we can apply a payment 'PENDING' and cancel it before it gets to this job and it becomes 'CANCELLED'. In the event of cancelled it was never PAID, we would just return the amount to the pending link amount to the short name.  The invoice will be still in APPROVED waiting to be paid again.

Maybe we need another status.

Reply:
After discussing, there is a check against invoice as well, so we should be fine. Resolving comment.